docs/06-contributors/999-rfcs/2819-stdlib.md

```diff
@@ -0,0 +1,115 @@
+# #2819 - Wing Utility Library
+
+- **Author(s):**: @eladb
+- **Submission Date**: 2023-06-07
+- **Stage**: Proposal
+- **Stage Date**: 2023-06-07
+
+> A specification for the Wing SDK utility libraries.
+
+## Design
+
+The Wing SDK includes a set of submodules which offer a set of useful functionality for building
+cloud applications.
+
+> This is just an initial spec and will likely change in the future.
+
+For each module, we show a list of function declarations. For each function, we specify which phase this function
+is bound to: `inflight` (inflight only - async), `preflight` (preflight only - synchronous) or
+`preflight inflight` (for phase-independent).
+
+## `util` module
```

Comment:
I wonder if we need a `math` module yet? eg

```js
preflight inflight min(arr: Set<num>): num
preflight inflight max(arr: Set<num>): num
preflight inflight abs(value: num): num
```

I'm fine with skipping this for now

Reply:
added a few